test/ad.jl

```diff
@@ -0,0 +1,22 @@
+
+using ReTest
```

Comment:
I'd prefer not using ReTest as it doesn't seem to be maintained anymore and we've run into other issues, e.g. https://github.com/TuringLang/AdvancedHMC.jl/issues/327.

Reply:
done!